components/Header.tsx

```diff
@@ -49,7 +56,7 @@ export const Header: React.FC = () => {
           <Navbar.Toggle aria-controls="basic-navbar-nav" />
           <Navbar.Collapse>
             <Nav>
-              <Nav.Link target="_blank" rel="noopener noreferrer" href="https://uio.edd.ca.gov">
+              <Nav.Link target="_blank" rel="noopener noreferrer" href={uioHomeLink}>
```

Comment:
Can Nav.Link not use the same URL Object syntax you used below? Just curious, would be nice to be consistent, doesn't really matter

Reply:
My hypothesis is that it can't because this is an external url whereas the language switcher is an internal url. Am I right? 🤞

Reply:
Frustratingly it cannot!

The URL Object syntax is another special super power we get from NextJS (which is where we get the [Link component from](https://nextjs.org/docs/api-reference/next/link)), that does not exist in Nav.Link (a [standard react bootstrap object](https://react-bootstrap.github.io/components/navs/#nav-link-props)). 

I spent a bit trying to find either a more robust way to pass the query in boostrap, or to replace this Nav.Link with a next Link component, but ultimately fell back to this method after a half hour of kicking at it with no success.